backend/utils.py

```
@@ -30,12 +29,11 @@
 REDIRECT_URI = "http://127.0.0.1:8000/auth/callback"
 
 
-print(os.getenv("google_password"))
+
 conf = ConnectionConfig(
-    MAIL_USERNAME="jamilahmediiuc@gmail.com",
+    MAIL_USERNAME= os.getenv("mymail"),
```

Ruff (0.8.2)

Use capitalized environment variable `MYMAIL` instead of `mymail`

(SIM112)